mysql-test/t/all_persisted_variables.test

```diff
@@ -43,9 +43,6 @@ call mtr.add_suppression("\\[Warning\\] .*MY-\\d+.* Changing innodb_extend_and_i
 call mtr.add_suppression("Failed to initialize TLS for channel: mysql_main");
 
 let $total_global_vars=`SELECT COUNT(*) FROM performance_schema.global_variables where variable_name NOT LIKE 'ndb_%' AND variable_name NOT LIKE 'debug_%'`;
-<<<<<<< HEAD
```

Comment:
Let's use the old version of the test:
[This](https://github.com/kamil-holubicki/percona-xtradb-cluster/commit/0d4521af859ee1bdedd56dffd7c5ff411d1a1cfe) is how I solved it in 8.4